lib/aur-repo

```diff
@@ -7,12 +7,21 @@ readonly AUR_COLOR=${AUR_COLOR-auto}
 # default arguments
 modifier=local
 vercmp_args=()
+vcs_only=0
 
-db_namever() {
-    awk '/%NAME%/ {
+db_parse() {
+    awk -v devel="$vcs_only" '/%NAME%/ {
         getline; pkgname=$1
-    } /%VERSION%/ {
-        getline; printf("%s\t%s\n", pkgname, $1)
+    }
+    /%BASE%/ {
+        getline; pkgbase=$1
+    }
+    /%VERSION%/ {
+        if (!devel ||
+            pkgbase ~ /-cvs$|-svn$|-git$|-hg$|-bzr$|-darcs$/ ||
```

Comment:
You could assign `pkgname` to `pkgbase` (or another variable) and only do one comparson.

Reply:
I am testing both `pkgname` and `pkgbase` to be closer to the implementation that was done in https://lists.archlinux.org/pipermail/aur-dev/2019-May/004649.html . Was keeping this wip commit as a placeholder for a proper equivalent implementation.

But testing only the `pkgname` could be enough. These approaches are nothing more than heuristics that may fail to determine vcs packages correctly anyway.

We could keep things simple, test only pkgname and allow users to customize the pattern in order to minimize false positives/negatives. i.e. keeping the `$AURVCS` idea, letting users to hardcode corner cases. Although https://github.com/AladW/aurutils/issues/511#issuecomment-508938979 suggests getting rid it.

Thoughts?

Reply:
Check just `pkgbase` for now as every package will have that.  Generally keep this easy to change because the real fix here is for the AUR RPC to report if a package is a VCS package as part of the info report.